github-helper.js

```diff
@@ -0,0 +1,66 @@
+exports.GithubHelper = class {
+  constructor({ client, context }) {
+    this.client = client;
+    this.context = context;
+  }
+
+  async assignToProjects({ columnName, newProjects }) {
+    const allProjects = await this.fetchProjects();
+    const columnIds = allProjects
+      .map((project) => project.node) // remove top-level "node" attribute
+      .filter(({ name: projectName }) => newProjects.includes(projectName))
+      .map(({ columns: { edges: columns } }) => (
+        columns.find((column) => column.node.name === columnName)
+          .node
+          .databaseId
+      ));
+    await this.addProjectCards({ columnIds });
+  }
+
+  async fetchProjects({ results, cursor } = { results: [] }) {
+    const query = `
+      query($cursor: String) {
+        repository(owner: "${this.context.repo.owner}", name: "${this.context.repo.repo}") {
+          projects(first: 100, after: $cursor) {
+            pageInfo {
+              hasNextPage
+              endCursor
+            }
+            edges {
+              node {
+                name
+                columns(first: 10) {
```

Comment:
please add a comment explaining why we are sure that we don't need to iterate here.